apps/meteor/client/sidebar/footer/voip/VoipFooter.tsx

```diff
@@ -59,6 +60,15 @@ export const VoipFooter = ({
 			  `
 			: '';
 
+	const deviceMenuOption = useDevicesMenuOption();
+
+	const options = useMemo(
+		() =>
+			deviceMenuOption && {
+				deviceSettings: deviceMenuOption,
+			},
+		[deviceMenuOption],
+	);
```

Comment:
Are we going to use the `useDevicesMenuOption` somewhere else? 
Because as I see we could have one `useDevicesMenusOptions` that does everything.